docs/conf.py

```diff
@@ -0,0 +1,57 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Bayesian Tracker (btrack) 🔬💻'
+copyright = '2022, Alan R Lowe'
+author = 'Alan R Lowe'
+
+# The full version, including alpha/beta/rc tags
+release = 'v0.4.2'
```

Comment:
is there a way to automate this from the latest tagged version on GitHub for example?